src/features/deposit/components/DepositForm/index.tsx

```diff
@@ -131,13 +126,21 @@ export const DepositForm = () => {
   }, [token, setValue])
 
   const balanceInsufficient = isInsufficientBalance(
-    watch("amount"),
+    amount,
     balance,
     nativeBalance,
     token,
     network
   )
 
+  const amountInputRef = useRef<HTMLInputElement | null>(null)
+
+  useEffect(() => {
+    if (network === BlockchainEnum.NEAR && amountInputRef.current) {
+      setTimeout(() => amountInputRef.current?.focus(), 0)
+    }
+  }, [network])
```

Comment:
Hard way to auto focus. Here are other options:
1. use `autoFocus` attribute
2. use function ref: `<input ref={(ref) => ref?.focus()} />`